ow_sim_tests/test/fps_monitor.test

```diff
@@ -3,10 +3,14 @@
 
     <arg name="gzclient" default="true"/>
 
-    <include file="$(find ow_sim_tests)/launch/atacama_y1a.launch" />
+    <include file="$(find ow)/launch/atacama_y1a.launch">
+        <arg name="gazebo_launch_file" default="$(find ow_sim_tests)/launch/empty_world_inject_fps_monitor_gui_plugin.launch"/>
+        <arg name="rqt_gui" value="false"/>
+        <arg name="use_rviz" value="false"/>
+    </include>
 
     <test test-name="minimum_fps" pkg="ow_sim_tests" type="test_minimum_fps.py">
-        <param name="min_fps" value="30.0"/>
+        <param name="min_fps" value="50.0"/>
```

Comment:
Actually, this increase may have caused my test failures.  What's the motivation?

Reply:
I always get 60 on my machine, so I thought I can make the test more strict. Do you know the average FPS that you get on your machine (when you have full view of atacama and the lander is standstill)?

Reply:
Ok, I have adjusted the **min_fps** to 45 based on the feedback from Terry that his old machine currently reports 47 fps.
I have also updated the failure message to report the observed frame rate.